apps/verify/views/paper-version-download.html

```diff
@@ -10,7 +10,7 @@ <h1 class="govuk-fieldset__heading">{{#t}}pages.paper-version-download.heading-1
   <p class="govuk-body">{{#t}}pages.paper-version-download.paragraph-3{{/t}}</p>
   <p class="govuk-body">{{#t}}pages.paper-version-download.paragraph-4{{/t}}</p>
 <div class="govuk-!-padding-bottom-3"></div>
-  <a href="offline-form" role="button" class="govuk-button govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-8">{{#t}}pages.paper-version-download.download-button{{/t}}</a>
+  <a href="prompt-sheet-for-working-offline" role="button" class="govuk-button govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-8">{{#t}}pages.paper-version-download.download-button{{/t}}</a>
```

Comment:
these classes look weird why is it `govuk-button--start`? and `govuk-!`

Reply:
Seems to be the convention used on the gov-uk-frontend-template css for margins and padding
![Screenshot 2019-09-18 at 12 01 07](https://user-images.githubusercontent.com/33282783/65143156-2f53e200-da0c-11e9-9501-e32ec8f0e9bb.png)

Reply:
These have a \